telcell/models/model.py

```diff
@@ -0,0 +1,39 @@
+from abc import abstractmethod
+
+from telcell.data.models import Track
+
+
+class Model:
+    """
+    Abstract base class for models that compute likelihood ratios for pairs of
+    tracks (`track_a` and `track_b`) under the hypotheses that the tracks are
+    colocated or dislocated.
+    """
+
+    @abstractmethod
```

Comment:
Niet dat ik enorm fan ben van ingewikkelde abstracte ontwerpen, maar `@abstractmethod` doet niet zoveel als de class waar ie in zit geen `ABC` in zn mro heeft toch?